docs/guide/types/ellipse.md

```diff
@@ -65,7 +65,12 @@ The following options are available for ellipse annotations.
 | [`borderWidth`](#styling) | `number`| Yes | `1`
 | [`borderDash`](#styling) | `number[]`| Yes | `[]`
 | [`borderDashOffset`](#styling) | `number`| Yes | `0`
+| [`borderShadowColor`](#styling) | [`Color`](../options#color) | Yes | `'transparent'`
 | [`backgroundColor`](#styling) | [`Color`](../options#color) | Yes | `options.color`
+| [`backgroundShadowColor`](#styling) | [`Color`](../options#color) | Yes | `'transparent'`
```

Comment:
wondering why background is after border here. would be ok to fix here if you want.

Reply:
All the options in the doc must be reordered because they are not sorted alphabetically. It was already in my TODO list.

Reply:
a specific PR is needed